test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -249,6 +249,22 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
             Assert.Equal("/Login?ReturnUrl=%2FHelloWorldWithAuth", response.Headers.Location.PathAndQuery);
         }
 
+        [Fact]
+        public async Task AuthorizePage_AllowAnonymousForSpecificPages()
+        {
+            // Arrange
+            var url = "/Admin/Login";
```

Comment:
@pranavkm don't worry I will take care about all of these, I'm already fixing them in my end. The main reason for that is after I download the latest bits, some packages are not restored probably, so I can't run the test in my end, hope I got a fix soon

thanks